tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -49,10 +49,22 @@ def allow_dirty(pytestconfig):
         "allow_dirty").lower() == "true" else False
 
 
+@pytest.fixture(scope='module')
+def enable_auth(pytestconfig):
+    return True if pytestconfig.getoption(
+        "enable_auth").lower() == "true" else False
+        
+        
 @pytest.fixture(scope='module')
 def client(core_url, serving_url, allow_dirty):
     # Get client for core and serving
     client = Client(core_url=core_url, serving_url=serving_url)
+    # if enable_auth is True, Google Id token will be 
+    # passed in the metadata for authentication. 
+    client = Client(core_url=core_url, 
```

Comment:
It's not clear why the Client is created twice here. Should the first one be removed?

Reply:
It should be.